alphagov/paas-cf
# [#117512517] Check postgres instances deployed via broker are available

## What

[#117512517 - Check postgres instances deployed via broker are available](https://www.pivotaltracker.com/story/show/117512517)

We want to know whether any changes to our environment affect the service instances and their binding to applications. For this we have deployed a health check application which serves a static endpoint, which replaces the existing health check application, and a database endpoint.

Pingdom health checks can now be set up to monitor the staging and production environments. This should result in the existing health check to be updated, and a new health check created for the database endpoint.

## How to review

* Deploy using `117512517_db_check_app`. 
	* Make sure `cf-deploy` passes.
	* Check The following end points in a browser:
	* `https://healthcheck.${DEPLOY_ENV}.dev.cloudpipelineapps.digital`
	* `https://healthcheck.${DEPLOY_ENV}.dev.cloudpipelineapps.digital/db`
* Set up the Pingdom checks for CI, staging, and production, using the instructions in the README. You can review the contents of the README in the process.
* Log into the Pingdom account and make sure there are checks for both end points for CI, staging and production. Instructions are in the README, which can also be reviewed for clarity.

## Who

Anyone but @henrytk or @combor
